.github/renovate.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "prHourlyLimit": 20,
+  "prConcurrentLimit": 20,
+  "recreateWhen": "always",
+  "enabledManagers": ["github-actions", "pre-commit", "composer", "dockerfile", "docker-compose"],
+  "extends": [
+    "github>alma/renovate:github-actions",
+    "github>alma/renovate:confidence-badges",
+    "github>alma/renovate:pre-commit"
+  ],
+  "dependencyDashboard": true,
+  "reviewers": ["team:squad-e-commerce-integrations"],
+  "baseBranches": ["develop"],
+  "packageRules": [
+    {
```

Comment:
I wonder if we need `composer` updates at all... it doesn't look like we're gonna approve any PR on this this side.

Reply:
FYI I added the exceptions for php & composer versions in dockerfile as for Prestashop.
But you also think that we should remove the manager "composer" ?